### ISSUE TITLE
Cannot use go to deifinition [ms-vscode.Go] provider FAILED

### ISSUE THREAD

**Issue by AndyCurry** [Microsoft/vscode-go#2420]
Hi All,
Recently I reinstalled vscode and tried to implement golang dev env.
Everything seems fine but the go to definition is no longer working.
Logs are included as below:

[2019-04-06 11:04:04.885] [exthost] [error] [ms-vscode.Go] provider FAILED
[2019-04-06 11:04:04.885] [exthost] [error] Error: spawn EPERM
	at ChildProcess.spawn (internal/child_process.js:357:11)
	at spawn (child_process.js:540:9)
	at Object.execFile (child_process.js:230:15)
	at Object.module.(anonymous function) [as execFile] (ELECTRON_ASAR.js:193:20)
	at Promise (c:\Users\aaron.cheng\.vscode\extensions\ms-vscode.go-0.9.2\out\src\goDeclaration.js:80:16)
	at new Promise (<anonymous>)
	at definitionLocation_godef (c:\Users\aaron.cheng\.vscode\extensions\ms-vscode.go-0.9.2\out\src\goDeclaration.js:74:12)
	at util_2.getGoVersion.then.goModules_1.getModFolderPath.then.modFolderPath (c:\Users\aaron.cheng\.vscode\extensions\ms-vscode.go-0.9.2\out\src\goDeclaration.js:38:24)

Thanks in advance.

**Comment by ramya-rao-a** [comment]
Looks like there are some permission issues here...

By default, the definition provider uses `godef` unless you have changed the `go.docsTool` setting.

Can you check where `godef` is installed? Try running `godef -h` in the integrated terminal. Does that succeed?

**Comment by AndyCurry** [comment]
@ramya-rao-a 
Thank you so much! Looks like it is the solution. 
I checked all the stuff under C:/GO/bin, and found most of the files here are not available. 
It says you must have read permission to view the properties of this object. 
The interesting part is that it says the onwer is me, but somehow I cannot execute it.
I added Users group with full control just in case ( Users group were not granted any permission).
Now it seems everything is running well.
Thanks again!
